ideascube/views.py

```diff
@@ -265,8 +267,6 @@ def get(self, *args, **kwargs):
 ajax_proxy = AjaxProxy.as_view()
 
 
-def javascript(request):
-    response = render(request, 'ideascube/includes/js.html',
-                      {'settings': settings})
-    response['Content-Type'] = 'text/javascript'
-    return response
+@cache_control(max_age=31536000)
```

Comment:
Huh, why did I not think of that?

Not only is it obvious, but there is another URL using a decorator from the same django module in exactly the same way a few lines above.

Thanks, fixed.